### PR TITLE
logger use milliseconds instead of seconds for reporting elapsed time

### LINKED ISSUE #5400: tool should use milliseconds not seconds for reporting elapsed time
These should all be in milliseconds, for instance:
```
Scanning project files...                            0.0s
Scanning package files...                            0.1s
Scanning asset files...                              0.0s
Scanning for deleted files...                        0.0s
No files to remove.
No files to update.
Performing hot reload...                             0.4s
Reloaded 31 of 421 libraries.
Reassembling application...                          0.6s
```

### PR DESCRIPTION
**Pull request by warent**
Fixes #5400

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/logger.dart [hunk 1]
```diff
@@ -250,8 +250,10 @@ class _AnsiStatus extends Status {
     live = false;
 
     if (showElapsedTime) {
-      double seconds = stopwatch.elapsedMilliseconds / 1000.0;
-      print('\b\b\b\b${seconds.toStringAsFixed(1)}s');
+      // Rendered undesirable in issue #5400
+      // double seconds = stopwatch.elapsedMilliseconds / 1000.0
```
Comment (abarth) [inline packages/flutter_tools/lib/src/base/logger.dart]: There's no reason to keep this commented out code.  We can just remove it.

**Comment by abarth** [comment]
LGTM

**Comment by abarth** [comment]
By the way, if you put the issue number in the body of the comment message as "Fixes #5400" (rather than in the title), then GitHub will automatically close the issue when the patch lands.

**Comment by warent** [comment]
Ah, that makes sense. Will do going forward, thanks!

**Pull request merged**